code/workspace/flipping-formats-and-cleanup/example_parameter.yaml

```diff
@@ -0,0 +1 @@
+# Convert the example JSON template into the YAML format with `cfn-flip`
```

Comment:
This file is redundant as it will get created when we run `cfn-flip example_parameter.json example_parameter.yaml`

Also, if the file exists it will prompt user to overwrite? I am removing the file.